### PR TITLE
FabArray::copy is deprecated, switch to MultiFab::Copy

### PR DESCRIPTION
**Pull request by maxpkatz**
## PR checklist

- [x] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

**Review by zingale — APPROVED** [review]

**Comment by zingale** [comment]
there are some crashes: http://groot.astro.sunysb.edu/Castro/test-suite/gfortran/2021-05-10-002/index.html

**Comment by zingale** [comment]
still some crashes: http://groot.astro.sunysb.edu/Castro/test-suite/gfortran/2021-05-10-003/index.html

File: Source/gravity/Gravity.cpp [hunk 1]
```diff
@@ -1241,7 +1241,8 @@ Gravity::average_fine_ec_onto_crse_ec(int level, int is_new)
 
     for (int n = 0; n < BL_SPACEDIM; ++n)
     {
-        grad_phi[level][n]->copy(*crse_gphi_fine[n], cgeom.periodicity());
+        MultiFab::Copy(*grad_phi[level][n], *crse_gphi_fine[n], 0, 0,
+                       crse_gphi_fine[n]->nComp(), 0);
```
[the remark this answers is not in the document]
Comment (WeiqunZhang) [reply Source/gravity/Gravity.cpp:1245]: Periodic boundary is involved because the variables are on the faces.

**Comment by maxpkatz** [comment]
I've made Weiqun's recommended change, please retest the failures.

**Comment by zingale** [comment]
we're good now: http://groot.astro.sunysb.edu/Castro/test-suite/gfortran/2021-05-10-004/index.html

**Pull request merged**